packages/playground/.eslintrc.cjs

```diff
@@ -87,6 +87,9 @@ module.exports = {
         project: "tsconfig.json",
       },
     },
+    "import/no-unresolved": {
+      ignore: ['^mml:']
+    },
```

Comment:
This just does not work for some reason... Which is why I've had to add the `/*eslint ... */` directive at the top of the type script files that import MML documents...